roles/rpm_ostree_uninstall_verify/tasks/main.yml

```diff
@@ -30,13 +30,13 @@
 
 - name: Set packages when deployment 0 is booted
   set_fact:
-    installed_pkgs: "{{ installed_json['deployments'][0]['packages'] }}"
-  when: installed_json['deployments'][0]['packages'] is defined and installed_json['deployments'][0]['booted']
+    installed_pkgs: "{{ installed_json['deployments'][0]['packages'] if installed_json['deployments'][0]['packages'] is defined else {} }}"
```

Comment:
Though note this should be fixed in v2017.4 (see https://github.com/projectatomic/rpm-ostree/pull/670). We did resping v2017.3 in Fedora at least to include that patch, though it's probably still present in some other streams. Maybe a small comment that we should be able to drop this hack once all the streams have v2017.4?